### ISSUE TITLE
--patch is not showing interactive output

### ISSUE THREAD

**Issue by anordal** [mystor/git-revise#120]
```
> git revise --interactive --patch    # Nothing happens, so I eventually press Ctrl+C
^CTraceback (most recent call last):
  File "/home/an/.local/bin/git-revise", line 8, in <module>
    sys.exit(main())
  File "/home/an/.local/lib/python3.8/site-packages/gitrevise/tui.py", line 263, in main
    inner_main(args, repo)
  File "/home/an/.local/lib/python3.8/site-packages/gitrevise/tui.py", line 233, in inner_main
    repo.git("add", "-p")
  File "/home/an/.local/lib/python3.8/site-packages/gitrevise/odb.py", line 229, in git
    prog = run(
  File "/usr/lib64/python3.8/subprocess.py", line 495, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/lib64/python3.8/subprocess.py", line 1015, in communicate
    stdout = self.stdout.read()
KeyboardInterrupt
```

This is with git-revise 0.7.0.
I get the same traceback without --interactive. However, --all works.

Running `git add --patch` works.

Thank you for making this wonderful tool! This problem won't stop me from using it routinely.

**Comment by anordal** [comment]
Turns out it's not hanging: It's just not showing the output. I can press y/n as usual to stage hunks in blind.

**Comment by anordal** [comment]
Fixed by #99.

Worth mentioning in the changelog? I should have thought about that in my PR as well, where I proposed the same fix (#120).
